src/components/Dialog/Dialog.st.css

```diff
@@ -57,6 +61,24 @@
     transform: none;
 }
 
+.root:mobile:notFullscreenMobile .outerContentWrapper {
+    display: flex;
+    align-items: center;
+    position:absolute;
+
+    top: value(NotFullscreenMobilePadding);
+    left: value(NotFullscreenMobilePadding);
+    width: calc(100% - 2 * value(NotFullscreenMobilePadding));
+    height: calc(100% - 2 * value(NotFullscreenMobilePadding));
+}
+
+.root:mobile:notFullscreenMobile .contentWrapper{
+    position: relative;
+    height: auto;
+    max-height: 440px;
+    min-width: auto;
+}
+
```

Comment:
We might want to update the inner padding on mobile mode.
Currently this is the padding value:
`.dialogContent {
  padding: 80px 32px;
}`

I will ask Michal (our designer about it).

Reply:
all right, will update if needed :)